mac/Keyman4MacIM/Keyman4MacIM/TextApiCompliance.m

```diff
@@ -177,10 +177,11 @@ - (BOOL)containedInHardCodedNoncompliantAppList:(NSString *)clientAppId {
                      [clientAppId isEqual: @"org.sil.app.builder.dictionary.DictionaryAppBuilder"] ||
                      //[clientAppId isEqual: @"com.microsoft.Word"] || // 2020-11-24[mcd]: Appears to work well in Word 16.43, disable legacy by default
                      [clientAppId isEqual: @"org.openoffice.script"] ||
-                     [clientAppId isEqual: @"com.adobe.illustrator"] ||
-                     [clientAppId isEqual: @"com.adobe.InDesign"] ||
-                     [clientAppId isEqual: @"com.adobe.Photoshop"] ||
-                     [clientAppId isEqual: @"com.adobe.AfterEffects"] ||
+                     // Adobe apps automatically detected as non-compliant
```

Comment:
Do we have an issue number for this?

Reply:
No, but I'll include the date in my comment to show when I removed these from the hard-coded list.